packages/multichain-transactions-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/multichain-transactions-controller",
-  "version": "0.2.0",
+  "version": "1.0.0",
```

Comment:
This one is still in early development, we want to keep it `<1.0.0` IMO, so `0.3.0` would be great.

Is that ok for you?

Reply:
yes changed